src/components/Menu/ReactSelectCustomComponents/Formatters/FormatOptionLabelWithLanguageLogo.tsx

```diff
@@ -22,37 +22,19 @@ export const FormatOptionLabelWithLanguageLogo = (languageOption: ExtendedReactS
       <div className="custom-svg-wrapper" style={{ transform: 'scale(0.833)' }}>
         <Component />
       </div>
-      {selectedOption === languageLabel && (
+      <div className="flex flex-row justify-between w-full">
+        <div className="py-4">{selectedOption !== languageLabel ? languageLabel : ''}</div>
         <div
-          className="text-active-orange"
+          className="text-active-orange p-4 font-medium text-center"
           style={{
             backgroundColor: '#FFF3EF',
-            fontSize: '0.6875rem',
+            fontSize: '0.688rem',
```

Comment:
<img width="387" alt="Screenshot 2024-08-21 at 09 56 17" src="https://github.com/user-attachments/assets/cc855caa-29fd-49c0-9f2f-049fac878fe0">
 Because it is so big, I don't think we have smaller than text-xs

Reply:
🤷  Magic number it is for now I guess